app/scripts/metamask-controller.js

```diff
@@ -1419,14 +1422,23 @@ module.exports = class MetamaskController extends EventEmitter {
   /**
    * A method for serving our ethereum provider over a given stream.
    * @param {*} outStream - The stream to provide over.
-   * @param {URL} senderUrl - The URI of the requesting resource.
-   * @param {string} extensionId - The id of the extension, if the requesting
-   * resource is an extension.
-   * @param {object} publicApi - The public API
-   */
-  setupProviderConnection (outStream, senderUrl, extensionId) {
-    const origin = senderUrl.hostname
-    const engine = this.setupProviderEngine(senderUrl, extensionId)
+   * @param {MessageSender} sender - The sender of the messages on this stream
+   * @param {boolean} internal - True if this is a connection with an internal process
+   */
+  setupProviderConnection (outStream, sender, internal) {
```

Comment:
nit: readability: `internal` -> `isInternal` (not sure to what extent we follow that convention, but 🤷‍♂ )

Reply:
Sure, that makes sense. Thanks